<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunDialog</class>
 <widget class="QDialog" name="RunDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run QualitySNPng</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Output directory:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="runName">
     <property name="inputMask">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Contig file:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="contigFileName">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Configuration:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="configurationComboBox">
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="runButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QPushButton" name="stopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QPlainTextEdit" name="logText">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="outputDirButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="editButton">
     <property name="text">
      <string>Edit...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>runName</tabstop>
  <tabstop>contigFileName</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>configurationComboBox</tabstop>
  <tabstop>runButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>logText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
